src/Datadog.Trace.ClrProfiler.Managed/Integrations/Types/ClrNames.cs

```diff
@@ -0,0 +1,24 @@
+namespace Datadog.Trace.ClrProfiler
+{
+    internal static class ClrNames
+    {
+        public const string Ignore = "_";
+
+        public const string Void = "System.Void";
+        public const string Object = "System.Object";
+        public const string Bool = "System.Boolean";
+        public const string String = "System.String";
+
+        public const string Int8 = "System.Int8";
+        public const string Int16 = "System.Int16";
+        public const string Int32 = "System.Int32";
+        public const string Int64 = "System.Int64";
+
+        public const string UInt8 = "System.UInt8";
+        public const string UInt16 = "System.UInt16";
+        public const string UInt32 = "System.UInt32";
+        public const string UInt64 = "System.UInt64";
+
+        public const string Byte = UInt8;
```

Comment:
I'm thinking since we are using the BCL names here (with the `System` namespace), we should change `UInt8` to `Byte` and `Int8` to `SByte`.